content/en/docs/configuration/acme/dns01/google.md

```diff
@@ -122,3 +130,104 @@ spec:
 ```
 
 For more details about `Certificates`, see [Usage](../../../../usage/).
+
+## GKE Workload Identity
+
+If you are deploying cert-manager into a [Google Container Engine (GKE)
+cluster](https://cloud.google.com/kubernetes-engine/) with workload identity
+enabled, you can leverage workload identity to avoid creating and managing
+static service account credentials. The [workload identity
+how-to](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
+provides more detail on how workload identity functions, but briefly workload
+identity allows you to link a Google service accounts (GSA) to Kubernetes
+service accounts (KSA). This GSA/KSA linking is two-way, i.e., you must
+establish the link in GCP _and_ Kubernetes. Once configured, workload identity
+allows Kubernetes pods running under a KSA to access the GCP APIs with the
+permissions of the linked GSA. The workload identity how-to also provides
+detailed instructions on how to enable workload identity in your GKE cluster.
+The instructions in the following sections assume you are deploying cert-manager
+to a GKE cluster with workload identity already enabled.
+
+### Link KSA to GSA in GCP
+
+The cert-manager component that needs to modify DNS records is the pod created
+as part of the cert-manager deployment. The [standard methods for deploying
+cert-manger to Kubernetes](../../../installation/kubernetes.md) create the
+cert-manager deployment in the cert-manager namespace and its pod spec specifies
+it runs under the cert-mananger service account. To link the GSA you created
+above to the cert-manager KSA in the cert-manager namespace in your GKE cluster,
+run the following command.
+
+```bash
+$ gcloud iam service-accounts add-iam-policy-binding \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:$PROJECT_ID.svc.id.goog[cert-manager/cert-manager]" \
+    dns01-solver@$PROJECT_ID.iam.gserviceaccount.com
+```
+
+If your cert-manager pods are running under a different service account, replace
+`goog[cert-manager/cert-manager]` with `goog[NAMESPACE/SERVICE_ACCOUNT]`, where
+`NAMESPACE` is the namespace of the service account and `SERVICE_ACCOUNT` is the
+name of the service account.
+
+### Link KSA to GSA in Kubernetes
+
+After deploying cert-manager, add the proper workload identity annotation to the
+cert-manager service account.
+
+```bash
+$ kubectl annotate serviceaccount --namespace=cert-manager cert-manager \
```

Comment:
Is there a way to do this with the Helm chart also?

Reply:
The helm chart offers the `serviceAccount.annotations` parameter for this purpose.

Reply:
Can we document that also for those who use helm?

Reply:
Yes, I added a blurb about that is my most recent push.

https://github.com/cert-manager/website/pull/306/files#diff-bafc869ae945b0b7647c4ca6a99a0b73R188-R190